packages/cf-component-modal/example/basic/component.js

```diff
@@ -55,11 +55,26 @@ class Component extends React.Component {
   }
 }
 
+class Application extends React.Component {
+  render() {
+    return (
+      <GatewayProvider>
+        <div>
+          {this.props.children}
+          <GatewayDest name="modal"/>
+        </div>
+      </GatewayProvider>
+    );
+  }
+}
```

Comment:
Oh shit wait– we need to use the non-universal form of react-gateway.

Hmm– I think I'll make a new version that supports the old way of rendering in additional to universal rendering.